queries.sql

```diff
@@ -42,4 +42,92 @@ GROUP BY name, neutered;
 SELECT species, MIN(weight_kg) AS min, MAX(weight_kg) AS max FROM animals GROUP BY species;
 SELECT species,AVG(escape_attempts) FROM animals 
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+SELECT animals.name
+FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT a.name
+FROM animals AS a
+JOIN species AS s ON a.species_id = s.id
+WHERE s.name = 'Pokemon';
+
+SELECT o.full_name, a.name
+FROM owners o
+LEFT JOIN animals a ON o.id = a.owner_id;
+
+SELECT s.name AS species_name, COUNT(*) AS animal_count
+FROM animals a
+JOIN species s ON a.species_id = s.id
+GROUP BY s.name;
+
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+JOIN species s ON a.species_id = s.id
+WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
+
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;
+
+SELECT o.full_name, COUNT(*) AS animal_count
+FROM owners o
+JOIN animals a ON o.id = a.owner_id
+GROUP BY o.full_name
+ORDER BY COUNT(*) DESC
+LIMIT 1;
+
+SELECT (SELECT name FROM animals WHERE id = visits.animal_id) AS animal , visit_date
+FROM visits
+WHERE vet_id = (SELECT id FROM vets WHERE name = 'William Tatcher')
+ORDER BY visit_date DESC
+LIMIT 1;
+
+SELECT COUNT(DISTINCT animal_id) AS animal_count
+FROM visits
+WHERE vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
+
+SELECT vets.name as vet , (SELECT name FROM species WHERE id = specializations.species_id) as species FROM vets 
+LEFT JOIN specializations ON specializations.vet_id =vets.id;
+
+SELECT (SELECT name FROM animals WHERE id = visits.animal_id) AS animal 
+FROM visits
+WHERE vet_id = (SELECT id FROM vets WHERE name = ' Stephanie Mendez')
+AND visits.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
```

Comment:
- [ ] Your query result as shown on your screenshot for this - `List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.` should show two results as highlighted below: 👇🏾 Please check your data or query command and correct accordingly.

<img width="434" alt="image" src="https://github.com/tjay1760/Vet-Clinic-Database/assets/84156714/2635367b-d65f-4f59-a33a-cfd89e946061">

Reply:
I have Updated the query 
![image](https://github.com/tjay1760/Vet-Clinic-Database/assets/52855706/4d5cb7b8-1cf0-4256-b90a-06dde8547378)